### PR TITLE
A1pr02_2

### PR DESCRIPTION
**Pull request by awanafiaz**
Incorporated all the changes discussed in the previous meeting, except the feedback form.

### CODE REVIEW

File: dataFilesEditor_manuallyOperated.Rmd [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+#generalData
+# Load file for Resource Table
+resource <- read_csv(file = here::here("data/ictr_table_resources.csv"))
+
+# Create the html version of links in resources
+## AwanEdit: added a column "identifier" to the resource table to use it as a merge key variable with the Data provided column entries in other tables
+resource_new <- resource %>% 
+  mutate(identifier = c("PDX", "HemOnc", "ngchm", "OncoMX", "MethCon5", "CIViC", "BioMuta", "TCIA", "SCGV", "ivyGlimpse", "TCPA", "dcmqi", "Region Templates", "Cistrome", "TOPAS", "TCGA", "1000 Genomes", "TARGET", "CCLE", "dbGAP", "CPTAC", "PheKB", "NSCLC Radiomics", "ICGC"))  %>%  relocate(identifier)
+
+write.csv(resource_new, file = "data/ictr_table_resources_new.csv")
```
Comment (carriewright11) [inline dataFilesEditor_manuallyOperated.Rmd]: I wonder if we could name this file something more informative than `_new` - maybe something about an identifier?

**Comment by awanafiaz** [comment]
List of changes included in this PR: 

1. Added a new banner image
2. Modified the homepage providing the description of the website and definitions of computing and data platforms
3. Moved all data.csv files to data folder to limit cluttering the main folder. 
    1. Created a new Rmd file (dataFilesEditor_manuallyOperated) to make changes to the data files manually using R, keeping original data files the same. This will allows us to make changes to the table information according to the responses from the developers, should the need arise. 
    2. Added (manually) a column called “identifier” to the resource_table.csv to set as the key variable for merging with other tables.
4. Merged Links from Resource table to the general, imaging, omics, clinical tables (with minor aesthetic modification to column specifications)
5. Added draft description to each individual table. 
6. Changed the website name to: ITCR Scientific Software Resources as instructed.
7. Changed the Data Resources menu icon to: fa-database
8. Added a new section “Feedback Form” for including the developer survey questionnaire.

File: index.Rmd [hunk 2]
```diff
@@ -1,44 +1,37 @@
...
+  text-justify: inter-word;
 }
 </style>
 
 <br> 
 
 ### **Welcome!**
 
-This website for ITCR Cancer Informatics Platform Database contains a comprehensive list of the bioinformatics platforms (computing and data) funded by **ITCR** *(Informatics Technology for Cancer Research)* grants. Each resource will have, along with their brief description, up to three links that navigate towards their -  
+This website for ITCR Scientific Software Resources contains a comprehensive list of the bioinformatics platforms (computing and data) that contain scientific software tools funded by **ITCR** *(Informatics Technology for Cancer Research)* grants.
```
Comment (carriewright11) [inline index.Rmd]:
```suggestion
This website is intended to help cancer researchers find bioinformatics computing platforms and data resources, many of which are funded by the ITCR (Informatics Technology for Cancer Research) program of the NCI (National Cancer Institute). Please note that some of the information presented in these tables came from the [ITCR website](https://itcr.cancer.gov/informatics-tools).
```

Comment (carriewright11) [reply index.Rmd]: @awanafiaz - you can commit suggestions from reviewers by pressing a button above if you would like to

Comment (carriewright11) [reply index.Rmd]: note that if you do - you would want to use `git pull`  (or pull on GitKraken) to get the remote changes if you want to add more commits from your local machine

File: index.Rmd [hunk 3]
```diff
@@ -1,44 +1,37 @@
...
+This website for ITCR Scientific Software Resources contains a comprehensive list of the bioinformatics platforms (computing and data) that contain scientific software tools funded by **ITCR** *(Informatics Technology for Cancer Research)* grants.  
 
-- main site  
-- pertinent documentation, and  
-- list of their publications.  
+  - **Computing resource platforms** are web or cloud-based platforms that provide computational support to researchers including, but not limited to, analysis, visualization, storage, and computation of large-scale bioinformatics data sets while facilitating efficient workflow,  database management, wider collaboration,  reproducibility, and streamlined scientific communication.  
   
-Please browse the through the resources on each page by using the navigation bar above. 
+  - **Data resource platforms** are remotely hosted sources of curated or collaborative bioinformatics data that researchers can access and utilize as required. The available data sets accommodate genomics, proteomics, imaging (cancel and radiology), as well as other types of data.
```
Comment (carriewright11) [inline index.Rmd]:
```suggestion
- **Data resources** are remotely hosted sources of curated or collaborative bioinformatics data that researchers can access and utilize as required. The available data sets accommodate genomics, proteomics, imaging (histology and radiology), as well as other types of data.
```

File: clinicalTable.Rmd [hunk 4]
```diff
@@ -5,12 +5,12 @@ output: html_document
 
 ### **Overview**
 
-This website contains a list of clinical informatics platforms funded by ITCR grants. Clinical platforms are those which work with several clinical data types, such as EMR Text and Clinical Notes. Each resource will have up to three links that navigate towards their main site, their documentation, and a list of their publications.
+The following table contains a list of clinical informatics platforms funded by ITCR grants. Clinical platforms work with several clinical data types, such as EMR Text and Clinical Notes, utilizing Natural Language Processing techniques, AI and machine learning algorithms in order to generate actionable insights with automated research tools. Note that, all resource summaries were taken from their respective sources.
```
Comment (carriewright11) [inline clinicalTable.Rmd]:
```suggestion
The following table contains a list of clinical informatics platforms. These platforms work with several types of clinical data, such as EMR Text and Clinical Notes, utilizing Natural Language Processing techniques, AI and machine learning algorithms in order to generate actionable insights with automated research tools. 
```

File: generalTable.Rmd [hunk 5]
```diff
@@ -3,14 +3,21 @@ title: "**General Informatics Table**"
...
+  body{
+  text-align: justify;
+  text-justify: inter-word;
+}
+</style>
+
 ### **Overview**
 
-This webpage contains a list of general informatics platforms funded by ITCR grants. General platforms are those which work with a wide variety of data types, from imaging data to “omics” data. Each resource will have up to three links that navigate towards their main site, their documentation, and a list of their publications.
+The following table contains a list of general informatics and biomedical platforms funded by ITCR grants. General platforms are those platforms which work with a wide variety of data types, from imaging data to “omics” data and provide computing resource support across different areas of bioinformatics research. Note that, all resource summaries were taken from their respective sources.
```
Comment (carriewright11) [inline generalTable.Rmd]:
```suggestion
The following table contains a list of general informatics and biomedical platforms that can help researchers work with publicly available data and or their own data. General platforms are those which work with a wide variety of data types, from imaging data to “omics” data, and provide computing resource support across different areas of bioinformatics research. 
```

File: imagingTable.Rmd [hunk 6]
```diff
@@ -5,12 +5,12 @@ output: html_document
 
 ### **About**
 
-This website contains a list of imaging informatics platforms funded by ITCR grants. Imaging platforms are those which work with several imaging data types, from pathology imaging to radiographic imaging. Each resource will have up to three links that navigate towards their main site, their documentation, and a list of their publications.
+The following table contains a list of imaging informatics platforms funded by ITCR grants. Imaging platforms work with several imaging data types, ranging from pathology imaging to radiographic imaging. These platforms allows researchers to create custom workflows, view and interact with analysis results, utilize imaging algorithms, analyze image tiles, as well as visualize and report analysis results. Note that, all resource summaries were taken from their respective sources.
```
Comment (carriewright11) [inline imagingTable.Rmd]:
```suggestion
The following table contains a list of imaging informatics platforms. These platforms work with several imaging data types, ranging from pathology imaging to radiographic imaging. These platforms allow researchers to create custom workflows, view and interact with analysis results, utilize imaging algorithms, as well as visualize and report analysis results. 
```

File: omicsTable.Rmd [hunk 7]
```diff
@@ -5,12 +5,12 @@ output: html_document
 
 ### **Overview**
 
-This website contains a list of "omics" informatics platforms funded by ITCR grants. "Omics" platforms are those which work with several genomic or proteomic data types, such as sequence data. Each resource will have up to three links that navigate towards their main site, their documentation, and a list of their publications.
+The following table contains a list of "omics" informatics platforms funded by ITCR grants. "Omics" platforms are those which work with several genomic or proteomic data types, such as sequencing, imaging, text and microarray data. These platforms allow access to interactively exploring and analyzing large-scale cancer data, create and share reproducible workspaces and results. The goal of these platforms include facilitating scientific collaboration and empowering the research community for genomic research. Note that, all resource summaries were taken from their respective sources.
```
Comment (carriewright11) [inline omicsTable.Rmd]:
```suggestion
The following table contains a list of "omics" informatics platforms funded by ITCR grants. "Omics" platforms are those which work with genomic or proteomic data types, such as RNA-Seq data. These platforms allow access to interactively explore and analyze large-scale cancer data, as well as create and share reproducible workspaces and results. The goals of these platforms are to facilitate scientific collaboration and to better empower the research community for "omic" research. 
```

File: resourceTable.Rmd [hunk 8]
```diff
@@ -5,12 +5,12 @@ output: html_document
 
 ### **Overview**
 
-This website contains a list of informatics resources funded by ITCR grants. Resources are remotely hosted sources of curated or collaborative data that researchers can use for projects. These resources also provide data to specific platforms highlighted within the various Platforms informatics tables.
+The following table contains a list of informatics resources funded by ITCR grants. Resources are remotely hosted sources of curated or collaborative data that are available to researchers. These platforms catalog common human genetic sequencing data, including genomic, epigenomic, transcriptomic, and proteomic data, cancer imaging data, radiology data, along with analytic and graphical tools to facilitate research endeavours. They also provide data to specific computing platforms which have been highlighted within the various computing resource tables. Note that, all resource summaries were taken from their respective sources.
```
Comment (carriewright11) [inline resourceTable.Rmd]:
```suggestion 
The following table contains a list of cancer data resources that are available to researchers. These resources include genomic, epigenomic, transcriptomic, proteomic, and cancer imaging data. 
```

File: dataFilesEditor_manuallyOperated.Rmd [hunk 9]
```diff
@@ -0,0 +1,45 @@
+---
+title: "**Data Files Editor**"
+output: html_document
+---
+
+### **Overview**
+
+Manually run the following r-chunks to make permanent edits to the data files into datafile_new.csv
```
Comment (carriewright11) [inline dataFilesEditor_manuallyOperated.Rmd]:
```suggestion
```

Comment (carriewright11) [reply dataFilesEditor_manuallyOperated.Rmd]: don't think we need this

File: dataFilesEditor_manuallyOperated.Rmd [hunk 10]
```diff
@@ -0,0 +1,45 @@
+---
+title: "**Data Files Editor**"
+output: html_document
+---
+
+### **Overview**
+
+Manually run the following r-chunks to make permanent edits to the data files into datafile_new.csv 
+
+## DO NOT knit this Markdown file.
```
Comment (carriewright11) [inline dataFilesEditor_manuallyOperated.Rmd]:
```suggestion
```

Comment (carriewright11) [reply dataFilesEditor_manuallyOperated.Rmd]: also don't need this

Comment (carriewright11) [reply dataFilesEditor_manuallyOperated.Rmd]: since we are creating a new file and have version control it's ok

**Review by carriewright11 — APPROVED** [review]
I have some changes to the descriptions and I think we can remove the information about manually running the code to add the identifier based on what we talked about in our meeting. Looks good to me if you make the suggested commits!

File: dataFilesEditor_manuallyOperated.Rmd (hunk 1 above)
Comment (awanafiaz) [reply dataFilesEditor_manuallyOperated.Rmd]: Agreed. I will rename it as "ictr_table_resources_identifier.csv" for better reference.

**Pull request merged**